pgdb/v1/fts.go

```diff
@@ -13,32 +12,14 @@ import (
 	"github.com/ductone/protoc-gen-pgdb/internal/stackoverflow"
 )
 
-var (
-	camelCaseRegex = regexp.MustCompile(`[[:upper:]][[:lower:]]+`)
-	// acronymRegex - this is to catch acronyms in a named group 'token'
-	// the suffix after the group is foward looking to avoid the first letter of a camel cased word
-	// for example given the text AuthSNSTest this regex captures SNS in token but matches on SNSTe:
-	//
-	// Given the text `AuthSNSTest`:
-	// 		?P<token>[[:upper:]]+) -> SNS - captured in "token"
-	//		[[:upper:]][[:lower:]] -> Te - discarded
-	// Given the text `AuthSNS Foo`:
-	// 		?P<token>[[:upper:]]+) -> SNS - captured in "token"
-	//		\s -> the trailing empty space ' ' - discarded
-	// Given the text `AuthSNS`:
-	// 		?P<token>[[:upper:]]+) -> SNS - captured in "token"
-	//		$ -> end of string
-	acronymRegex = regexp.MustCompile(`(?P<token>[[:upper:]]+)([[:upper:]][[:lower:]]|$|\s)`)
-)
-
-const template = "$token"
-
 type SearchContent struct {
 	Type   FieldOptions_FullTextType
 	Weight FieldOptions_FullTextWeight
 	Value  interface{}
 }
 
+const MinWordSize = 3
```

Comment:
shouldn't be exported

Reply:
changed in 87da3bca39151e296b78e05bdbc51cd0e7d6099e